ts/components/Alert/index.tsx

```diff
@@ -97,45 +124,125 @@ export const Alert = ({
+
+  // Scaling transformation applied when the button is pressed
+  const animationScaleValue = IOScaleValues?.magnifiedButton?.pressedState;
+
+  // Using a spring-based animation for our interpolations
+  const progressPressed = useDerivedValue(() =>
+    withSpring(isPressed.value, IOSpringValues.button)
+  );
+
+  // Interpolate animation values from `isPressed` values
+  const pressedAnimationStyle = useAnimatedStyle(() => {
+    // Scale down button slightly when pressed
+    const scale = interpolate(
+      progressPressed.value,
+      [0, 1],
+      [1, animationScaleValue],
+      Extrapolate.CLAMP
+    );
+
+    return {
+      transform: [{ scale }]
+    };
+  });
+
+  const onPressIn = useCallback(() => {
+    // eslint-disable-next-line functional/immutable-data
+    isPressed.value = 1;
+  }, [isPressed]);
+  const onPressOut = useCallback(() => {
+    // eslint-disable-next-line functional/immutable-data
+    isPressed.value = 0;
+  }, [isPressed]);
+
+  const renderMainBlock = () => (
+    <>
+      <Icon
+        name={mapVariantStates[variant].icon}
+        size={iconSize}
         color={mapVariantStates[variant].foreground}
-        weight={"Regular"}
-        accessibilityRole="text"
-      >
-        {content}
-      </Label>
-      {action && (
-        <>
-          <VSpacer size={8} />
-          <ButtonLink color={variant} onPress={onPress} label={action} />
-        </>
-      )}
+      />
+      <HSpacer />
+      <View style={IOStyles.flex}>
+        {title && (
+          <>
+            <NewH4 color={mapVariantStates[variant].foreground}>{title}</NewH4>
+            <VSpacer size={8} />
+          </>
+        )}
+        <Label
+          color={mapVariantStates[variant].foreground}
+          weight={"Regular"}
+          accessibilityRole="text"
+        >
+          {content}
+        </Label>
+        {action && (
+          <>
+            <VSpacer size={8} />
+            <Text
+              style={[
+                isDesignSystemEnabled ? styles.label : styles.labelLegacy,
+                { color: IOColors[mapVariantStates[variant].foreground] }
+              ]}
+              numberOfLines={1}
+              ellipsizeMode="tail"
+            >
+              {action}
+            </Text>
+          </>
+        )}
+      </View>
+    </>
+  );
+
+  const StaticComponent = () => (
+    <View
+      ref={viewRef}
+      style={[
+        styles.container,
+        fullWidth ? styles.spacingFullWidth : styles.spacingDefault,
+        { backgroundColor: IOColors[mapVariantStates[variant].background] }
+      ]}
+      testID={testID}
+      accessible={false}
+      accessibilityRole="alert"
+      accessibilityHint={accessibilityHint}
+    >
+      {renderMainBlock()}
     </View>
-  </View>
-);
+  );
+
+  const PressableButton = () => (
+    <Pressable
+      ref={viewRef}
+      testID={testID}
+      onPress={onPress}
+      onPressIn={onPressIn}
+      onPressOut={onPressOut}
```

Comment:
I think that the component looks good, but as we know, can happen that in some cases (eg. open a bottom-sheet) the `Pressable` component doesn't trigger the `onPressOut` event, in this case it is good if you add also the `onTouchEnd` event 

 ```suggestion
       onPressOut={onPressOut}
       onTouchEnd={onPressOut}
```